README.md

```diff
@@ -33,9 +35,36 @@ changelist scientific-python/changelist v0.2.0 main
 
 This will list all pull requests, authors and reviewers that touched commits
 between `v0.2.0` and `main` (excluding `v0.2.0`).
-Pull requests are sorted into section according to the configuration in
+Pull requests are sorted into sections according to the configuration in
 `tool.changelist.label_section_map`.
 
+## Writing pull request summaries
+
+By default, changelist will fall back to the title of a pull request and its
+GitHub labels to sort it into the appropriate section. However, if you want
+longer summaries of your changes you can add a code block with the following
+form anywhere in the description of the pull request:
```

Comment:
@stefanv, re https://github.com/scientific-python/changelist/pull/53#pullrequestreview-1846584015: yes, overriding the title is documented.